index.html

```diff
@@ -9,238 +9,291 @@
+        </div>
+        <section>
+            <div class="z">
+                <h2 class="p">About me</h2>
+                <p class="hello">Hello I’m a software developer! I can help you build a product , feature or website
+                    Look
+                    through some of my
+                    work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                    contact
+                    me.</p>
+                <section class="abouted">
+                    <div class="proyect2">
+                        <button type="button" class="see">Get My Resume</button>
+                        <img class="frame" src="pictures/Frame1.png" alt="Frame">
+                    </div>
+                    <img class="line" src="pictures/vector4.png" alt="line">
+
+                </section>
+                <section class="experience">
+                    <div class="language">
+                        <div class="rectangle">
+                            <img class="mob" src="pictures/Rectangle.png" alt="rectangle">
+                            <h5 class="letras2">Lenguage</h5>
+                        </div>
+                        <ul class="agrupador">
+                            <li>JavaScript</li>
+                            <li>Ruby</li>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                        </ul>
+                    </div>
+                    <div class="language">
+                        <div class="rectangle">
+                            <img class="mob" src="pictures/rectangle1.png" alt="rectangle">
+                            <h5 class="letras2">Frameworks</h5>
+                        </div>
+                        <ul class="agrupador">
+
+                            <li>Bootstrap</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSPec</li>
+                            <li>CapyBara</li>
+                            <li>Selenium</li>
+                        </ul>
+                    </div>
+                    <div class="language">
+                        <div class="rectangle">
+                            <img class="" src="pictures/circle.png" alt="rectangle">
+                            <h5 class="skills">Skills</h5>
+                        </div>
+                        <ul>
+                            <li>Codekit</li>
+                            <li>Github</li>
+                            <li>Codepen</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </div>
+                    <footer>
+                        <h2 class="interested">I'm always interested in hearing about new projects, so if you'd like to
+                            chat please get in touch.</h2>
+
+
+                        <!-- Name input -->
+                        <form action="https://formspree.io/f/mwkyoykw" method="post">
```

Comment:
- [ ] Let's design the `form` design as in the Figma website.
- The text `I'm always interested in.....` should be aligned in middle. Let's Fix this.
- The border line below the `name` & `email` input should be present.
- The border line above the icons should be present. Let's add this.
- Your website 
![ergerg](https://user-images.githubusercontent.com/26767383/155212481-b9119fc9-8554-4127-bdc5-94eed0fed3d0.PNG)

- Figma website
![gngfng](https://user-images.githubusercontent.com/26767383/155212515-2ff340cf-e85c-4b3a-a429-344421b753b7.PNG)